static/templates/topic_edit_form.handlebars

```diff
@@ -1,10 +1,10 @@
 {{! Client-side Mustache template for rendering the topic edit form. }}
 
 <form id="topic_edit_form" class="form-horizontal">
-{{!        <div class="controls edit-controls"> }}
-          <input type="text" value="" class="inline_topic_edit header-v" id="inline_topic_edit" />
-            <button type="button" class="topic_edit_save primary"><i class="icon-vector-ok"></i></button>
-            <button type="button" class="topic_edit_cancel primary"><i class="icon-vector-remove"></i></button>
-{{!        </div> }}
+    {{!        <div class="controls edit-controls"> }}
```

Comment:
We should probably just remove these comments manually as part of fixing this template.